test/utils/IexecWrapper.ts

```diff
@@ -104,7 +104,7 @@ export class IexecWrapper {
                 this.accounts.anyone,
             ).workerpool_stake_ratio()
         ).toNumber();
-        return ((workerpoolPrice * stakeRatio) / 100) * volume;
+        return Math.floor((workerpoolPrice * stakeRatio) / 100) * volume;
```

Comment:
Why this change now ?

Reply:
We need to mimic what is done in the contract, workerpool price is not always divisible without remainder when applying 30% to get staking ratio